assessments/guide/security-reviewer.md

```diff
@@ -52,14 +52,41 @@ guidelines for conflict of interest situations are important to prevent:
-A reviewer with a soft conflict is required to document the nature of the conflict in the
-security assessment documentation. It is the responsibility of the security reviewer
-to make known his/her own conflict of interests.
-
-For each project, 2 SIG-Security chairs must sign off on the conflicts presented to them that the assessment lead has no conflicts, and reviewers have no hard conflicts. Reasons for accepting and rejecting conflicts should be documented.
+The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts:
+* A hard conflict makes a reviewer ineligible to review a project.
+* A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
+* It is not unusual for reviewers to have soft conflicts. The diversity of reviewers that are familiar with a project and can provide a deeper insight together with a fresh set of eyes is beneficial to the success of a security review.
+
+All reviewers must provide a conflict declaration to indicate which hard or soft conflicts do, or do not exist 
+when they volunteer to be a reviewer.  This is done by placing a comment on the issue associated with the 
+assessment.
+Should a conflict arise during the time of the assessment, reviewers should notify the lead security reviewer when 
+they become aware of the potential conflict, so the new conflict may be consulted with the facilitator and/or chairs.
+
+The lead security reviewer has the responsibility of ensuring a balanced review,
+and as part of that before kicking off the assessment must:
+* Check that all reviewers have conflict-of-interest declarations,
+* Provide their own declaration of any potential conflict-of-interest (or lack thereof),
+* Ask clarifying questions, if needed
+* Assert that:
+  1. They have reviewed all conflict-of-interest declarations from the reviewers;
+  2. There are no hard-conflicts present without a waiver;
+  3. They believe that the review team is able to provide a balanced and fair assessment
+
+Update the above assertion if a new conflict-of-interest becomes known during
+the course of the assessment.
+
+The Security Assessment Facilitator or a SIG-Security chair must review the 
+Lead Security Reviewer conflict-of-interest assertion.
+
+If there any conflicts are presented, then 2 SIG-Security chairs must approve
+the security review team. Reasons for accepting and rejecting conflicts should
+be documented.
+In most cases, the existence of a hard conflict will prevent a SIG member from participating in the assessment for 
+which their hard conflict exists.  Depending on the circumstances of the particular conflict, the assessment, and the 
+may project, the chairs and the assessment facilitator may determine if the hard conflict may be waived.  Should this 
+occur, the decision's justification will be documented to ensure it clearly depicts the circumstances for granting the 
+waiver.
```

Comment:
I think the 2 chair requirement should perhaps be for this?

Reply:
corrected the wording, somehow it was miswritten, let me know if this works @lumjjb